### PR TITLE
Calendar cross links

### PR DESCRIPTION
**Pull request by noahmanger**
## Summary

Adds styles for the "reporting deadlines" links:
image

Includes a new mix-in for generating icons in circles. 

## Screenshots

Styleguide:
image

cc @jenniferthibault

### CODE REVIEW

**Comment by jenniferthibault** [comment]
One question: Why does this show up as a headline style in the style guide, while in the wild it's regular copy sized? Any way to adjust the style guide rendering?

File: scss/components/_options.scss [hunk 1]
```diff
@@ -17,6 +17,9 @@
 //         <img src="img/guide--candidates.jpg" alt="Federal Election Commission Campaign Guide: Congressional Candidates and Committees, June 2014">
 //       </a>
 //       <a class="t-sans" href="http://www.fec.gov/pdf/candgui.pdf">(PDF)</a>
+//       <div class="icon-heading--calendar--secondary option__aside__alt-action">
+//         <a href="#">Reporting deadlines</a>
```
Comment (jenniferthibault) [inline scss/components/_options.scss]: Is this really setting the text here? I would have expected it in the CMS repo

Comment (noahmanger) [inline scss/components/_options.scss]: Nope! This is just the example for the style guide.

**Comment by noahmanger** [comment]
@jenniferthibault this class isn't setting anything about the typography. It's just adding the calendar icon in a circle. That way it can be used on any type of text element.

**Comment by jenniferthibault** [comment]
👍

**Pull request merged**